users/user_test.go

```diff
@@ -0,0 +1,90 @@
+package users
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/LF-Engineering/dev-analytics-libraries/users/mocks"
+	"github.com/stretchr/testify/assert"
+
+	json "github.com/json-iterator/go"
+)
+
+const (
+	OKStatus = 200
+)
+
+var (
+	httpClientProvider    = &mocks.HTTPClientProvider{}
+	auth0ClientProvider   = &mocks.Auth0ClientProvider{}
+	elasticClientProvider = &mocks.ESClientProvider{}
+	slackClientProvider   = &mocks.SlackProvider{}
+	userStruct            = &Usr{
+		os.Getenv("PLATFORM_USER_SERVICE_ENDPOINT"),
```

Comment:
you may use static values, cuz tests should be independent from environment

Reply:
yes, but I don't want to put any URLs here, so I will keep this "as-is"

Reply:
you already use the mocked version of elasticProvider, so you will not depend on these values you cam leave them empty but avoid depending on real stuff like envVars while unit testing

Reply:
OK, but then we also need to fix "orgs" package - this one is based on that. I won't fix "orgs" one in this PR.

Reply:
Correct! do it in the both please

Reply:
Done, also added `mockery --all --case=underscore` as a `make mocks` sub command.